### ISSUE TITLE
SQLAlchemy 1.4 Column.copy() Deprecation Warning

### ISSUE THREAD

**Issue by samuelchodur** [LANL-Seismoacoustics/pisces#44]
SQLAlchemy 1.4 includes [deprecation warnings](https://github.com/sqlalchemy/sqlalchemy/issues/5953) for the `Column.copy()` method. This method is used heavily in [`pisces/schema/kbcore.py`](https://github.com/LANL-Seismoacoustics/pisces/blob/master/pisces/schema/kbcore.py) and [`pisces/schema/css3.py`](https://github.com/LANL-Seismoacoustics/pisces/blob/master/pisces/schema/css3.py). The simplest fix I can think of at this time would be to change from `copy() `to `_copy()`, although I am not sure if this is appropriate for the context. Seems too easy to be a good solution :grinning:. 

Could this be a good opportunity to develop something addressing #23?

**Comment by guyskk** [comment]
I not recommend use the private `_copy` method. 

In my case I use `Column.copy` to define snapshot table:

```python
def snapshot_table_of(source_table: sa.Table) -> sa.Table:
    column_s = [col.copy() for col in source_table.columns]
    table_name = source_table.name + '_snapshot'
    snapshot_table = sa.Table(table_name, *column_s)
    return snapshot_table
```

The `Table.tometadata` method mentioned in [changelog](https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html#change-4835edbe9953b4005da244bb252da030) not suitable for me, and I'm not sure how to replace the deprecated method.

Is stdlib`copy.deepcopy` the right solution?

**Comment by jkmacc-LANL** [comment]
Closed by #49.

These are useful comments, so let's leave this Issue open for now.  Thanks for your thoughts, all!

**Comment by jkmacc-LANL** [comment]
We're also working on a solution to this that uses `deepcopy`.  Closing for now.
